subsys/bluetooth/mesh/Kconfig

```diff
@@ -52,13 +52,41 @@ config BT_MESH_UNPROV_BEACON_INT
 	  This option specifies the interval (in seconds) at which the
 	  device sends unprovisioned beacon.
 
+if BT_MESH_PB_ADV
```

Comment:
Would it make sense to hide PB-ADV parameters under menuconfig?

Reply:
Since this parameters only valid for PB-ADV.